docs/src/paradox/01-introduction/standoff-rdf.md

```diff
@@ -0,0 +1,72 @@
+<!---
+Copyright © 2015-2018 the contributors (see Contributors.md).
+
+This file is part of Knora.
+
+Knora is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Knora is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public
+License along with Knora.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+# Standoff/RDF Text Markup
+
+@@toc
+
+[Standoff markup](http://uahost.uantwerpen.be/lse/index.php/lexicon/markup-standoff/)
+is text markup that is stored separately from the content it describes. Knora's
+Standoff/RDF markup stores content as a simple Unicode string, and represents markup
+separately as RDF data. This approach has some advantages over commonly used markup systems
+such as XML:
+
+- XML and other hierarchical markup systems assume that a document is a hierarchy, and
+  have difficulty representing
+  [non-hierarchical structures](http://www.tei-c.org/release/doc/tei-p5-doc/en/html/NH.html)
+  or multiple overlapping hierarchies. Standoff markup can easily represent these structures.
+- Markup languages are typically designed to be used in text files. But there is no
+  standard system for searching and linking together many different text files containing
+  markup. By storing markup as RDF, Knora can search for markup structures
```

Comment:
There is http://exist-db.org/exist/apps/homepage/index.html 

However, I do not know how they handle links to entities defined in other places, like a link to a person defined in a global index.

Reply:
I said “there is no standard system”. ExistDB is not a standard.